modules/Sources/Dependencies/SDKSynchronizer/SDKSynchronizerLive.swift

```diff
@@ -29,7 +29,7 @@ extension SDKSynchronizerClient {
             spendParamsURL: databaseFiles.spendParamsURLFor(network),
             outputParamsURL: databaseFiles.outputParamsURLFor(network),
             saplingParamsSourceURL: SaplingParamsSourceURL.default,
-            loggingPolicy: .noLogging
+            loggingPolicy: .default(.debug)
```

Comment:
[non-blocking] Is this change intentional?

Reply:
It is not, thanks for the catch.